specification/legacy.md

```diff
@@ -85,52 +81,54 @@ in a group column of a participants.tsv file:
 ]
 ```
 
-If `"Edges"` is not present in the model,
-then a linear sequence is assumed from the first `Node` to the last,
-with no filters specified on each `Edge`.
+If `"Edges"` is not present in the model, then a linear sequence is assumed from
+the first `Node` to the last, with no filters specified on each `Edge`.
 
 ## Inputs
 
 ### Specifying image inputs
 
-Specification of image inputs to a BIDS Model is deliberately minimalistic.
-The spec itself says nothing about how images are to be passed in as inputs to the first node in a model spec;
-this is deliberately left as an implementation detail for BIDS Model-compliant software.
-To ensure independence of statistical modeling from preprocessing,
-the spec also does not constrain what _kind_ of images can be passed as inputs.
-It does, however, allow any inputs passed in to be filtered on the basis of keywords
-(or entities) defined in the core BIDS spec.
-Minimally, this includes the `"task"`, `"run"`, `"session"`, and `"subject"` keywords
-(additional keywords may be added as they’re merged into the BIDS spec).
-For example:
+Specification of image inputs to a BIDS Model is deliberately minimalistic. The
+spec itself says nothing about how images are to be passed in as inputs to the
+first node in a model spec; this is deliberately left as an implementation
+detail for BIDS Model-compliant software. To ensure independence of statistical
+modeling from preprocessing, the spec also does not constrain what _kind_ of
+images can be passed as inputs. It does, however, allow any inputs passed in to
+be filtered on the basis of keywords (or entities) defined in the core BIDS
+spec. Minimally, this includes the `"task"`, `"run"`, `"session"`, and
+`"subject"` keywords (additional keywords may be added as they’re merged into
+the BIDS spec). For example:
 
 ```yaml
 {
   "Name": "my_analysis",
   "BIDSModelVersion": "1.0.0",
-  "Input": {
-    "task": "motor",
-    "run": 1
-  },
+  "Input": { "task": "motor", "run": 1 },
```

Comment:
prettier seems to have reformatted some of those too on top of the word wrapping